az-reportal/src/test/java/azkaban/reportal/util/tableau/CountDownTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.reportal.util.tableau;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Duration;
+import org.junit.Test;
+
+public class CountDownTest {
+
+  @Test
+  public void testMoreTimeRemaining() throws InterruptedException {
```

Comment:
Unit tests usually finish very soon, within 1-2 seconds. Is it possible to reduce the runtime of this test using some mocking method?

Reply:
A good unit test should finish within milliseconds. 
@jamiesjc
Thanks for paying attention to test running time.

Reply:
thanks, updated